### DISCUSSION TITLE
Trackpoint support?

### DISCUSSION THREAD

**Discussion by laur89** [joshreve/dactyl-keyboard#101] [Ideas]
Hi!
Would you consider adding trackpoint module support to this version of dactyl?

**Comment by bosd** [comment]
I'm interested in this too.
Did you happen to integrate one?
I'm looking for a way to `hack` this into the thumb cluster area.

From my experience, thumbs are underused on the keyboard. And it is way easier to accuratly control the trackpointer with a thumb then an index finger.
